src/utils/date.ts

```diff
@@ -3,19 +3,28 @@ const [, offsetHours] = new Date(2022, 0, 1).toISOString().split('T')
 export const formatDate = (
     date: string | Date,
     locale = 'de-CH',
-    timeZone = 'Europe/Zurich'
+    timeZone = 'Europe/Zurich',
+    withTime = false,
```

Comment:
Wäre wahrscheinlich weniger komplex, wenn du hier eine formatDateTime Funktion erstellen würdest. Evtl kannst du gemeinsame Optionen in einer Konstante definieren um alles DRY zu halten. Was meinst du?